java/vortex-jni/src/main/java/dev/vortex/api/expressions/proto/TemporalMetadatas.java

```diff
@@ -0,0 +1,89 @@
+/**
+ * (c) Copyright 2025 SpiralDB Inc. All rights reserved.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package dev.vortex.api.expressions.proto;
+
+import com.google.common.base.Preconditions;
+import java.io.ByteArrayOutputStream;
+import java.nio.charset.StandardCharsets;
+import java.util.Optional;
+
+@SuppressWarnings("MutablePublicArray")
+public final class TemporalMetadatas {
+    private TemporalMetadatas() {}
+
+    public static byte TIME_UNIT_NANOS = 0;
+    public static byte TIME_UNIT_MICROS = 1;
+    public static byte TIME_UNIT_MILLIS = 2;
+    public static byte TIME_UNIT_SECONDS = 3;
+    public static byte TIME_UNIT_DAYS = 4;
+
+    public static final byte[] DATE_DAYS = new byte[] {TIME_UNIT_DAYS};
```

Comment:
Do these actually need to be public?

Reply:
They're used in the proto serde, so i guess technically this class can be kept package-private for now?